sdk/include/platform/sunburst/platform-uart.hh

```diff
@@ -0,0 +1,70 @@
+#pragma once
+#pragma push_macro("CHERIOT_PLATFORM_CUSTOM_UART")
+#define CHERIOT_PLATFORM_CUSTOM_UART
+#include_next <platform-uart.hh>
+#pragma pop_macro("CHERIOT_PLATFORM_CUSTOM_UART")
+
+/**
+ * OpenTitan UART
+ */
+template<unsigned DefaultBaudRate = 115'200>
+class OpenTitanUart
+{
+	public:
+	uint32_t intrState;
+	uint32_t intrEnable;
+	uint32_t intrTest;
+	uint32_t alertTest;
+	uint32_t ctrl;
+	uint32_t status;
+	uint32_t rData;
+	uint32_t wData;
+	uint32_t fifoCtrl;
+	uint32_t fifoStatus;
+	uint32_t ovrd;
+	uint32_t val;
+	uint32_t timeoutCtrl;
```

Comment:
These should all have doc comments explaining what they do (which are for reading, which for writing, which bits are valid in each and what they mean, and so on).